app/cli.py

```diff
@@ -63,9 +70,146 @@ def seed_test_db():
 
             db.session.commit()
 
+    @app.cli.command("update_datasets")
+    def update_datasets():
+        from app import db
+        from app.models import Dataset
+        from sqlalchemy.exc import IntegrityError
+
+        logger.setLevel(app.logger.getEffectiveLevel())
+
+        data_path = app.config.get("DATA_PATH", "static/data/projects").lstrip("/")
+        search_path = Path(app.root_path).joinpath(data_path)
+
+        for project in search_path.glob("*"):
+            if not project.is_dir():
+                continue
+
+            datalad_conf = project.joinpath(".datalad/config")
+            dataset_id = _read_config(datalad_conf, "datalad \"dataset\"",
+                                      "id")
+
+            git_conf = project.joinpath(".git/config")
+            annex_uuid = _read_config(git_conf, "annex", "uuid")
+
+            model = _get_dats_model(project)
+            if model:
+                description = model.get("description")
+                try:
+                    download_path = model.get("storedIn").get("name")
+                except AttributeError:
+                    download_path = None
+                name = model.get('title')
+                if model.get("types"):
+                    modality = ",".join([item.get("value").lower()
+                                        for item in model.get("types")
+                                        if item.get("value")])
+                else:
+                    modality = None
+                version = model.get("version")
+                try:
+                    raw_data_url = model.get("distributions").get("access").get(
+                            "landingPage")
+                except AttributeError:
+                    raw_data_url = None
+                data_format = _get_formats(model)
+                private = model.get("privacy") == "Open Access"
+            else:
+                description = None
+                download_path = None
+                raw_data_url = None
+                name = None
+                modality = None
+                version = None
+                data_format = None
+                is_private = None
+
+            record = Dataset(dataset_id=dataset_id,
+                             annex_uuid=annex_uuid,
+                             description=description,
+                             owner_id=None,
```

Comment:
So owner is not in the DATS? right?

Reply:
There's a field that looks right, but the current database schema expects an integer so I assumed it was meant to link to an entry in the user table (there's no foreign key constraint on that column though so I could be wrong)